### PR TITLE
feat(flake): expose hie-bios

### PR DESCRIPTION
**Pull request by teto**
since it is helpful to help troubleshoot hie.yaml issues and since the binaries are built anyway.

<a href="https://gitpod.io/#https://github.com/haskell/haskell-language-server/pull/2221"></a>

### CODE REVIEW

**Review by jneira — APPROVED** [review]
lgtm!

**Comment by berberman** [comment]
Thanks, that's a good point. Can we add hie-bios to the buildInputs of dev shell (I assume it should be used in command line), so that we don't need maintain the attributes like hie-bios-901 for each GHC version?

**Comment by teto** [comment]
@berberman like this ?

File: flake.nix [hunk 1]
```diff
@@ -246,6 +247,11 @@
           haskell-language-server-8107 = mkExe ghc8107;
           haskell-language-server-901 = mkExe ghc901;
 
+          # hie-bios packages
+          hie-bios-884 = ghc884.hie-bios;
+          hie-bios-8107 = ghc8107.hie-bios;
+          hie-bios-901 = ghc901.hie-bios;
+
```
Comment (berberman) [inline flake.nix:254]:
```suggestion
```

**Comment by jneira** [comment]
@berberman thanks for reviewing it, good to merge now?

**Pull request merged**